drivers/volume/csi/csi.go

```diff
@@ -549,10 +547,6 @@ func (c *csi) GetBackupStatus(backup *storkapi.ApplicationBackup) ([]*storkapi.A
 	vsClassMap := make(map[string]*kSnapshotv1beta1.VolumeSnapshotClass)
 	for _, vInfo := range backup.Status.Volumes {
 		if vInfo.DriverName != storkCSIDriverName {
-			continue
-		}
-		// Once a snapshot has been backed up and cleaned up, skip it
-		if vInfo.Status == storkapi.ApplicationBackupStatusSuccessful {
 			volumeInfos = append(volumeInfos, vInfo)
```

Comment:
If it is not a CSI driver, we do not need to append the volume to the volumeInfos. We can just have continue alone.

Reply:
fixed this